libutil/dr_frontend.h

```diff
@@ -95,6 +95,11 @@ typedef int ssize_t;
 
 #ifdef WINDOWS
 #    include <tchar.h>
+#    ifdef _UNICODE
+#        define _dr_nlen wcsnlen
```

Comment:
We should not use a leading underscore in the global scope: that's reserved.  I would say, follow the lead of all the other wide-vs-narrow generic names which are following the Windows standard: use _tcsnlen, which is already defined for Windows.  Just add it for UNIX.